cmd/state/stateless.go

```diff
@@ -67,18 +65,13 @@ func runBlock(tds *state.TrieDbState, dbstate *state.Stateless, chainConfig *par
 	if err := statedb.CommitBlock(ctx, dbstate); err != nil {
 		return fmt.Errorf("commiting block %d failed: %v", block.NumberU64(), err)
 	}
-	if err := dbstate.CheckRoot(header.Root, checkRoot); err != nil {
```

Comment:
Remove `tds` from the parameters of the `runBlock` function, it is not needed anymore